packages/blockchain-wallet-v4-frontend/src/modals/CoinifyExchangeData/Payment/selectors.js

```diff
@@ -8,7 +8,7 @@ export const getData = createDeepEqualSelector(
     selectors.core.data.coinify.getLimits,
     selectors.core.data.coinify.getLevel,
     selectors.core.data.coinify.getMediums,
-    selectors.core.data.coinify.getKycs
+    selectors.core.data.coinify.getSortedKycs
```

Comment:
Are there 2 selectors for kycs? Do you ever need them unsorted?

Reply:
Yeah, leftover from old code. I'll remove the one that's not needed.